README.md

```diff
@@ -35,12 +52,33 @@ mvn archetype:generate \
 -DgroupId=com.bsiag.demo  \
 -DartifactId=tasks \
 -Dversion=1.0-SNAPSHOT
+```
 
+```
 cd tasks
+```
 
 ### Build standard Project with Maven and execute
+
+```
 mvn clean package
-java -jar target/tasks-\*.jar
+java -jar target/tasks-1.0-SNAPSHOT.jar
+```
+
+### Play with the Application
+
+In your browser change to [localhost](http://localhost:8080). This starts the UI of the Tasks application. To log in you may use any of the following pre-created users (user/password) below.
+
+* root/eclipse
+* alice/test
+* bob/test
+
+In case your user has rights to access the aAPI of the pplications API (log in with root/eclipse to check for this) you can also access the business objects (tasks) via REST.
```

Comment:
There is a small typo here: The 'a' of application slid before the API.
Also, 'API' seems to be doubled by accident.